### DISCUSSION TITLE
DBeaver 24.1.2 - Possible Bug - The Data Transfer pop-up window is maximized beyond my screen edges.

### DISCUSSION THREAD

**Discussion by scokna4gh** [dbeaver/dbeaver#34730] [DBeaver Development] [answered]
In DBeaver 24.1.2, when I try to export data from the data editor, the Data Transfer pop-up window is maximized beyond my screen edges. Here is a screen capture video to show what is happening: https://app.screencast.com/JxUbMsfKEjo7j

In the previous version of DBeaver, the Data Transfer pop-up window opened smaller than the current DBeaver window.

**Comment by E1izabeth** [comment, marked as answer]
Thank you for bug report. Here is the issue about it https://github.com/dbeaver/dbeaver/issues/34705

**Reply by scokna4gh** [reply]
Thanks!
